### ISSUE TITLE
can we get the information_schema tables and comments to match dbt docs with a DRY approach?

### ISSUE THREAD

**Issue by amychen1776** [allen-wong-tech/AssetManagement#1]
select table_type object_type, table_name object_name, comment
    from information_schema.tables
    where table_schema = 'PUBLIC' and comment is not null
        union all
    select 'COLUMN' object_type, table_name || '.' || column_name object_type, comment
    from information_schema.columns
    where table_schema = 'PUBLIC' and comment is not null
    order by 1,2;

**Comment by amychen1776** [comment]
@allen-wong-tech Could you explain a little more about how users will use the comments on the information schema in their workplace?

**Comment by allen-wong-tech** [comment]
Yes, in Financial Services users need to know what a column's formula means.  ie if they're using Present Value or some complicated formula, users need to know those details since they're making trading decisions on it.  So, the idea is that they would pull the comment column into the BI tool of their choice.  These comments can be added in the Create or Alter Table statement or via the Comment command.  For [example](https://github.com/Snowflake-Labs/sfguide-financial-asset-management/blob/master/setup/finserv%20demo%2030%20DDL.sql#L74-L75).
